tests/fixtures/playlist.nml

```diff
@@ -11,7 +11,29 @@
             </ENTRY>
           </PLAYLIST>
         </NODE>
+        <NODE TYPE="FOLDER" NAME="Foo">
+          <SUBNODES COUNT="2">
+            <NODE TYPE="PLAYLIST" NAME="Preparation2">
+              <PLAYLIST ENTRIES="212" TYPE="LIST" UUID="f80b2bdebbb843eb897c1ae90af896f6">
+                <ENTRY>
+                  <PRIMARYKEY TYPE="TRACK" KEY="osx/:Users/:johndoe/:Music/:pechundschwefel/:549043_The_Sky_Was_Pink_Icelandic_Version.mp3"/>
+                </ENTRY>
+              </PLAYLIST>
+            </NODE>
+            <NODE TYPE="FOLDER" NAME="Bar">
+              <SUBNODES COUNT="1">
+                <NODE TYPE="PLAYLIST" NAME="Preparation3">
+                  <PLAYLIST ENTRIES="212" TYPE="LIST" UUID="f80b2bdebbb843eb897c1ae90af896f7">
+                    <ENTRY>
+                      <PRIMARYKEY TYPE="TRACK" KEY="osx/:Users/:johndoe/:Music/:pechundschwefel/:549043_The_Sky_Was_Pink_Icelandic_Version.mp3"/>
+                    </ENTRY>
+                  </PLAYLIST>
+                </NODE>
+              </SUBNODES>
+            </NODE>
+          </SUBNODES>
+        </NODE>
       </SUBNODES>
     </NODE>
   </PLAYLISTS>
-</NML>
+</NML>
```

Comment:
newline is missing

Reply:
A little confused on this one.  I added a newline but not sure if that's what you were asking for.

Reply:
Thats the POSIX convention, see here for explanation: https://stackoverflow.com/a/729795/319905

> 
> Because that’s how the POSIX standard defines a line:
> 
>     3.206 Line
>         A sequence of zero or more non- <newline> characters plus a terminating <newline> character.
> 
> Therefore, lines not ending in a newline character aren't considered actual lines. That's why some programs have problems processing the last line of a file if it isn't newline terminated.